common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
@rabbah wanted your inputs to make suggested changes around pluggability. 
Is there already work going on around this(you had mentioned come prototype under progress) which I need to wait and integrate with or is this something which can be done independently on similar lines as you and @ddragosd had discussed in previous comments?
I want to make sure changes I make are aligned to suggestions here.